src/clients/ntp-client_proposal.rb

```diff
@@ -362,10 +351,10 @@ def Write(param)
       return :success if param["write_only"]
 
       # One-time adjusment without running the ntp daemon
-      # Meanwhile, ntpdate was replaced by sntp
+      # Meanwhile, ntpdate was replaced by sntp and later by chrony
       ntpdate_only = param["ntpdate_only"]
```

Comment:
I'd remove this assignment. Its name is misleading. It is used like twenty lines lower and at one place only.